.github/workflows/publish-vectors-pages.yaml

```diff
@@ -0,0 +1,44 @@
+# Simple workflow for deploying static content to GitHub Pages
+name: Publish Test Vectors
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Single deploy job since we're just deploying
+  deploy:
+    environment:
+      name: github-pages
+      # TODO - need to set up a custom domain to host the web5 vectors. cannot reuse tbdex.dev in multiple repos
```

Comment:
@finn-tbd in the meantime, can we host both test harness output and the vectors on the same default domain you're using already?

Reply:
fine by me